docs/blog/posts/2023-04-20-introducing-trusted-publishers.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Introducing 'Trusted Publishers'
+description: Announcing a new, more secure way to publish to PyPI
+author: Dustin Ingram
+publish_date: 2024-04-20
+tags:
+  - publishing
+  - security
+  - oidc
+---
+
+Starting today, PyPI maintainers can adopt a new, more secure publishing method
+that does not require long-lived passwords or API tokens to be shared with
+external systems.
+
+### About trusted publishing
+
+"Trusted publishing" is our term for using the [OpenID Connect
+(OIDC)] standard to exchange short-lived identity tokens between a trusted
+third-party service and PyPI. This method can be used in automated environments
+and eliminates the need to use username/password combinations or manually
+generated API tokens to authenticate with PyPI when publishing.
+
+Instead, PyPI maintainers can configure PyPI to trust an identity provided by a
+given OpenID Connect Identity Provider (IdP). This allows allows PyPI to verify
+and delegate trust to that identity, which is used to request short-lived,
+tightly-scoped API tokens from PyPI. These API tokens never need to be stored
+or shared, rotate automatically by expiring quickly, and provide a verifiable
+link between a published package and its source.
+
+### Using trusted publishing with GitHub Actions
+
+PyPI currently supports trusted publishing with GitHub Actions, using [their
+support for OpenID Connect].
+
+After configuring PyPI to trust a given GitHub repository and workflow, users
+of the PyPA's ['pypi-publish' GitHub Action] can quickly and easily adopt
```

Comment:
I think @webknjaz has stated a preference for linking to the marketplace and using the name of the action there instead of the GitHub repositoriy.

(This is also how we get better search results for that term 😉)

Reply:
Makes sense!